### ISSUE TITLE
[Server Issue] Player kicked from Server when harvesting wheat 7.2.5

### ISSUE THREAD

**Issue by fisele** [StrangeLoopGames/EcoIssues#6156]
**Steps to reproduce:** 
1. Harvest Wheat

**Expected behavior:**
Wheat appears in inventory.

**Actual behavior:**
Server throws an exception, the harvesting player is disconnected.

**Further information:**
If the inventory of the player is full, nothing happens.

**Host system:**
I7 4770
32GB RAM
Windows Server 2016 Datacentre Edition
250GB SSD

**Dump:**

```
`--BEGIN DUMP--
Dump Time
03/16/2018 20:33:58

Exception
System.Exception: Caught exception invoking RPC PlayerInteract on Player!

Der angegebene Schlüssel war nicht im Wörterbuch angegeben.


Stack:
   bei System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   bei LiteDB.IndexService.AddNode(CollectionIndex index, BsonValue key, Byte level, IndexNode last)
   bei LiteDB.IndexService.AddNode(CollectionIndex index, BsonValue key, IndexNode last)
   bei LiteDB.LiteEngine.InsertDocument(CollectionPage col, BsonDocument doc, BsonType autoId)
   bei LiteDB.LiteEngine.<>c__DisplayClass22_0.<Insert>b__0(CollectionPage col)
   bei LiteDB.LiteEngine.Transaction[T](String collection, Boolean addIfNotExists, Func`2 action)
   bei LiteDB.LiteEngine.Insert(String collection, BsonDocument doc, BsonType autoId)
   bei LiteDB.LiteCollection`1.Insert(T document)
   bei Eco.Core.Utils.AtomicAction.SimpleAtomicAction.TryApply()
   bei Eco.Core.Utils.AtomicAction.MultiAtomicAction.TryApply()
   bei Eco.Core.Utils.AtomicAction.MultiAtomicAction.TryApply()
   bei Eco.Gameplay.Plants.PlantEntity.TryHarvest(Player player, Boolean picked)
   bei Eco.Mods.TechTree.ScytheItem.OnActLeft(InteractionContext context)
   bei Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   bei Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)

   bei Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   bei Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   bei Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   bei Eco.Shared.Networking.NetPort.ProcessBuffer()
   bei Eco.Shared.Networking.NetPort.Update()
   bei Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()

--END DUMP--`
```

**Comment by fisele** [comment]
Happening with wolves as well.

**Comment by MarjieAVolk** [comment]
Hm, I am hoping this will be fixed in 7.3 (because we have updated our third party database software).

In the mean time, you can probably continue playing by deleting your .db file.  This will clear your stats (which will clear all graphs and reset any action counts that you were referencing for laws), but shouldn't affect your gameplay too much.

**Comment by MarjieAVolk** [comment]
Going to close as fixed in 7.3, I have some amount of confidence :P  Let me know if you do see it again though.
